### PR TITLE
Add dtale

### PR DESCRIPTION
**Pull request by Juanlu001**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Juanlu001** [comment]
Classic, license is not packaged.

**Comment by Juanlu001** [comment]
> [noarch_python] Noarch package contains binary script: dtale.exe
> ##[error]Cmd.exe exited with code '1'.

I guess packages with scripts can't be `noarch` then

**Comment by Juanlu001** [comment]
@conda-forge-admin, please ping team

**Comment by chrisburr** [comment]
> > [noarch_python] Noarch package contains binary script: dtale.exe
> > ##[error]Cmd.exe exited with code '1'.
> 
> I guess packages with scripts can't be `noarch` then

You can, you just need to declare the console_scripts to conda:
* https://github.com/man-group/dtale/blob/master/setup.py#L106
* https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

File: recipes/dtale/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - dtale
+    - dtale.cli
+
+about:
+  home: https://github.com/man-group/dtale/
+  license: LGPL
+  license_file: LICENSE
+  summary: 'D-Tale is the combination of a Flask back-end and a React front-end to bring you an easy way to view & analyze Pandas data structures'
+
+  # The remaining entries in this section are optional, but recommended.
```
Comment (chrisburr) [inline recipes/dtale/meta.yaml]:
```suggestion
```

File: recipes/dtale/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - Flask
+    - future
+    - itsdangerous
+    - pandas
+    - requests
+    - scipy
+    - six
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
```
Comment (chrisburr) [inline recipes/dtale/meta.yaml]: Can you remove these comments and add a test for the `dtale` command?

File: recipes/dtale/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  summary: 'D-Tale is the combination of a Flask back-end and a React front-end to bring you an easy way to view & analyze Pandas data structures'
+
+  # The remaining entries in this section are optional, but recommended.
+  doc_url: https://dtale.readthedocs.io
+  dev_url: https://github.com/man-group/dtale/
+
+extra:
+  recipe-maintainers:
+    - Juanlu001
+    - AlexS12
```
Comment (chrisburr) [inline recipes/dtale/meta.yaml]: @AlexS12 Are you happy to be a maintainer of this?

As @aschonfeld offered to also be listed as a maintainer in https://github.com/man-group/dtale/pull/60#issuecomment-585521032 do you want to add them as well?:
```suggestion
    - AlexS12
    - aschonfeld
```

Comment (Juanlu001) [reply recipes/dtale/meta.yaml]: Oops, forgot! Applying

**Comment by Juanlu001** [comment]
Thanks for the review @chrisburr ! I addressed all the comments.

**Pull request merged**